### PR TITLE
Feature/leaf 1271 log primary admin

### PR DESCRIPTION
**Pull request by g-theriot**

### CODE REVIEW

**Comment by dastan-rahimi** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Clones added
============
- LEAF_Request_Portal/Inbox.php  1
- LEAF_Request_Portal/form.php  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/VALEAF/LEAF/pullRequest?prid=5443207&bid=17849112)

**Comment by dastan-rahimi** [comment]
- History is not tracking when you unset a Primary admin

- No pagination 

- Dialog too wide
Screen Shot 2020-05-01 at 9 27 45 AM

File: LEAF_Request_Portal/form.php [hunk 1]
```diff
@@ -1623,7 +1623,7 @@ public function hasDependencyAccess($dependencyID, $details)
                  else{
                     $empUID = $this->getEmpUID($resPerson[0]['userID']);
                                                                 
-                    return $this->checkUserAccess($empUID);
```
Comment (thomasnguyen95) [inline LEAF_Request_Portal/form.php:1626]: I believe this function has been renamed before, is this being changed back?

File: LEAF_Request_Portal/admin/templates/mod_groups.tpl [hunk 2]
```diff
@@ -351,21 +353,46 @@ function viewHistory(groupID){
...
         type: 'GET',
-        url: 'ajaxIndex.php?a=gethistory&type=group&id='+groupID,
+        url: 'ajaxIndex.php?a=gethistory&type='+type+'&id='+groupID,
         dataType: 'text',
         success: function(res) {
             dialog_simple.setContent(res);
             dialog_simple.indicateIdle();
             dialog_simple.show();
+
+            if(type == "primaryAdmin"){
```
Comment (thomasnguyen95) [inline LEAF_Request_Portal/admin/templates/mod_groups.tpl:368]: NP: indentation

**Comment by dastan-rahimi** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           

Clones added
============
- LEAF_Request_Portal/Inbox.php  1
- LEAF_Request_Portal/form.php  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/VALEAF/LEAF/pullRequest?prid=5443207&bid=17849112)

**Comment by dastan-rahimi** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           

Clones added
============
- LEAF_Request_Portal/Inbox.php  1
- LEAF_Request_Portal/form.php  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/VALEAF/LEAF/pullRequest?prid=5443207&bid=17849112)

**Pull request merged**